app/src/main/java/ar/com/wolox/android/training/model/User.java

```diff
@@ -0,0 +1,55 @@
+package ar.com.wolox.android.training.model;
+
+
+public class User {
+    private Integer id;
+    private String username;
+    private String email;
+    private String password;
+    private String picture;
+    private String cover;
+    private String description;
+    private String location;
+    private String name;
+    private String phone;
+
```

Comment:
Constructor must be defined too. I like to create the empty one and with all the fields , but that it's up to you :)